packages/utilities/psammead-styles/src/colours.js

```diff
@@ -11,7 +11,6 @@ export const C_RHINO = '#5A5A5A';
 export const C_STONE = '#D5D0CD';
 export const C_CHALK = '#ECEAE7';
 export const C_SHADOW = '#3F3F42';
-export const C_CLOUD_DARK = '#757575';
```

Comment:
We shouldn't remove Cloud Dark from our colour palette. It should be available and marked as `legacy` colour in Storybook. This is because as we update our application over time, we will want to ensure we have reference to past colours.